dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -64,16 +65,60 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
         }
     }
 
+    void RegisterUnpackagedApplicationHelper(
+        const winrt::guid& remoteId,
+        _Out_ wil::unique_cotaskmem_string &unpackagedAppUserModelId)
+    {
+        auto coInitialize = wil::CoInitializeEx();
```

Comment:
@danielayala94 Why do we need this coinitialize call at all? Am I missing something here?

Reply:
At the very least, I think it requires a comment if there is a gotcha.

Reply:
Are we seeing API calls fail for STA?

Reply:
About the STA test, I'm taking this as part of this bug to be addressed next week -> #1372